mediawiki/LocalSettings.php

```diff
@@ -158,7 +167,6 @@
 $wgDefaultUserOptions['wikieditor-preview'] = 1;
 #$wgDefaultUserOptions['wikieditor-publish'] = 1;
 
-require_once "$IP/extensions/Scribunto/Scribunto.php";
 $wgScribuntoDefaultEngine = 'luastandalone';
```

Comment:
I thought about these two lines and the `chmod a+x` line in `install_extensions`, I need to investigate some more to find out what the default mode of that file is + what the default values for these two variables are.

I will do this investigation later and create a separate PR! I agree, it would be nice to clean-up 😅